idaes/surrogate/sampling/scaling.py

```diff
@@ -13,6 +13,24 @@
 import pandas as pd
 
 class OffsetScaler(object):
+
+    @staticmethod
+    def create_normalizing_scaler(dataframe):
+        """
+        Creates a scaling object that normalizes the data between 0 and 1
+
+        Args:
+           dataframe: pandas DataFrame
+              The dataframe containing the data (usually the training data)
+              that will be used to compute the scaling factor and offset
+        """
+        expected_columns = list(dataframe.columns)
+        offset = dataframe.min()
+        factor = dataframe.max()-dataframe.min()
+        print(offset)
```

Comment:
Should these still be here, or were they just for debugging? If we want to keep them, should we use a logger instead of a print?